README.md

```diff
@@ -1,6 +1,48 @@
 # PSR - Log Util
 Utilities for the [`psr/log`][] package that backs the [PSR-3][] specification.
 
+ * `Psr\Log\Test\LoggerInterfaceTest` provides a base test class for ensuring compliance with the LoggerInterface.
+ * `Psr\Log\Test\TestLogger` is a mock class for testing purposes.
+
+## Installation
+
+This package should be used only for tests.
+
+For PHP 8.0+, you may remove support for older versions of `psr/log`:
+
+```json
+{
+  "require": {
+    "php": ">=8.0",
+    "psr/log": "^3.0"
+  },
+  "require-dev": {
+    "psr/log-utils": "^1.1"
+  }
+}
+```
+
+If the project supports older versions of PHP, you can accept all versions:
```

Comment:
I would add a small note saying that having a concrete class support all 3 of them is not so straightforward.